src/main/java/codeu/controller/AdminServlet.java

```
@@ -156,6 +157,13 @@ void appendUser(String userName) {
               UUID.nameUUIDFromBytes(userName.getBytes()), userName, "password", Instant.now());
       userStore.addUser(user);
       currentUser = user;
+      try {
+        if (currentConversation != null) {
+          membersInConvo.add(UUID.nameUUIDFromBytes(userName.getBytes()));
+        }
+      } catch (Exception m) {
+        m.printStackTrace();
```

Can we propagate this exception instead? Silently handling exceptions is a bit of a risk